src/types/mod.rs

```diff
@@ -1804,27 +1831,33 @@ impl ChainParametersFamily for ChainParameterVersion1 {
     type Output = ChainParametersV1;
 }
 
+impl ChainParametersFamily for ChainParameterVersion2 {
+    type Output = ChainParametersV2;
+}
+
 pub type ChainParameters<CPV> = <CPV as ChainParametersFamily>::Output;
 
 #[derive(Debug, SerdeSerialize, SerdeDeserialize)]
 #[serde(rename_all = "camelCase")]
 /// Values of reward parameters.
-pub struct RewardParametersSkeleton<MD> {
+pub struct RewardParametersSkeleton<MD, GR> {
```

Comment:
It'd be useful to comment on what GR and MD are so we can better maintain this.

Alternatively name the type parameters in a more verbose way.

Reply:
I have made the names of the generics more explicit and added a bit to the documentation